avocado/utils/tb.py

```diff
@@ -0,0 +1,39 @@
+"""
+Traceback standard module plus some additional APIs.
```

Comment:
This is not a issue inside the `avocado.utils.backtrace module`, but yes, it clashes with traceback on stdlib. Maybe `stacktrace` or `traceback_info`?